### ISSUE TITLE
Prompt and variable files are not always relative to configuration directory

### ISSUE THREAD

**Issue by emsi** [promptfoo/promptfoo#372]
When a prompt is declared like:

```
prompts:
  file://prompts/java.json
```
It's relative to the configuration directory, however when following syntax is used:
```
prompts:
  prompts/java.json: java
```
The path is not treated as relative to configuration path.

On the other hand when variable is loaded from file using `file://` syntax:
```
tests:
  - description: Cached Property
    vars:
      prompt: file://prompts/1_cachedproperty.txt
```
that file path is not treated as relative to configuration file.

This inconsistent behavior makes it difficult to tun promptfoo with multiple configuration file.

**Comment by typpo** [comment]
Thanks for the report!  Fix is in #375, which should consistently resolve paths relative to the configuration path.

**Comment by typpo** [comment]
Fix published in 0.35.0
